README.md

```diff
@@ -207,7 +207,7 @@ bundle update
     gemrat --pessimistic cocoapods cocoapods-check xcpretty gym deliver match
     bundle update
     ```
-- If you get a message from match saying _Could not create another certificate, reached the maximum number of available certificates._ see this [StackOverflow answer](http://stackoverflow.com/a/26780411/822249)
+- If you get a message from match saying _Could not create another certificate, reached the maximum number of available certificates._ see this [explanation](docs/manual-match.md)
```

Comment:
`...you might need to to use the existing development and appstore certificates. See [docs/manual-match.md](docs/manual-match.md) for more information and instructions.`